src/app/Instance/BridgeOverview/Components/BOSourceList.tsx

```diff
@@ -69,14 +72,18 @@ export const BOSourceList = (props: BOSourceListProps): JSX.Element => {
             </TextContent>
           </StackItem>
           <StackItem>
-            <ClipboardCopy
-              isReadOnly
-              hoverTip="Copy"
-              clickTip="Copied"
-              variant={ClipboardCopyVariant.expansion}
-            >
-              {bridgeIngressEndpoint}
-            </ClipboardCopy>
+            {bridgeIngressEndpoint ? (
+              <ClipboardCopy
+                isReadOnly
+                hoverTip="Copy"
+                clickTip="Copied"
+                variant={ClipboardCopyVariant.expansion}
+              >
+                {bridgeIngressEndpoint}
+              </ClipboardCopy>
+            ) : (
+              <Skeleton width="100%" />
+            )}
```

Comment:
Can you set the `bridgeIngressEndpoint` attribute in our stories? I can see skeleton in BridgeOverview stories.

Reply:
Yeah sure.